homeassistant/components/lutron.py

```diff
@@ -105,3 +113,45 @@ def name(self):
     def should_poll(self):
         """No polling needed."""
         return False
+
+
+class LutronButton:
+    """Representation of a button on a Lutron keypad.
+
+    This is responsible for firing events as keypad buttons are pressed
+    (and possibly released, depending on the button type). It is not
+    represented as an entity; it simply fires events.
+    """
+
+    def __init__(self, hass, keypad, button):
+        """Register callback for activity on the button."""
+        self._name = '{}: {}'.format(keypad.name, button.name)
```

Comment:
`self._name` isn't used as an instance attribute. We can just store it as a local variable.